### ISSUE TITLE
bookmark button at the top of the timetable tab has a different accessibility invocation.

### ISSUE THREAD

**Issue by Pluu** [DroidKaigi/conference-app-2023#497]
Wrong accessibility is called for some components in timetable tab.
The bookmark button at the top should have the correct accessibility called.

- Current Behavior: Timetable Accessibility
- Expected behavior: Bookmark accessibility

![Screenshot_20230815_014752](https://github.com/DroidKaigi/conference-app-2023/assets/1534926/48305d80-a4f8-4fde-b1be-d04e706d26a7)

**Comment by takahirom** [comment]
Thanks for finding this!

**Comment by Pluu** [comment]
🙋

**Comment by takahirom** [comment]
Assigned!
